maths/germain_primes.py

```diff
@@ -0,0 +1,46 @@
+"""
+A Germain prime is any prime p, where 2p + 1 is also prime.
```

Comment:
```suggestion
A Sophie Germain prime is any prime p, where 2p + 1 is also prime.
```
Wikipedia seems to call them "Sophie Germain primes", with the first and last name of Sophie Germain.